contracts/test/MockEzEthPool.sol

```diff
@@ -0,0 +1,247 @@
+    }
+
+    function getBufferedEther() external pure returns (uint256) {
+        return 0;
+    }
+
+    function getTotalPooledEther() public view returns (uint256) {
+        return totalPooledEther + _getAccruedInterest();
+    }
+
+    function getTotalShares() public view returns (uint256) {
+        return totalShares;
+    }
+
+    function sharesOf(address _account) external view returns (uint256) {
+        uint256 tokenBalance = balanceOf[_account];
+        return tokenBalance.mulDivDown(getTotalShares(), getTotalPooledEther());
+    }
+
+    /// Mock ///
+
+    function setRate(uint256 _rate_) external requiresAuthDuringCompetition {
+        _accrue();
+        _rate = _rate_;
+    }
+
+    function getRate() external view returns (uint256) {
+        return _rate;
+    }
+
+    function _accrue() internal {
+        uint256 interest = _getAccruedInterest();
+        if (interest > 0) {
+            totalPooledEther += interest;
+        }
+        _lastUpdated = block.timestamp;
+    }
+
+    function _getAccruedInterest() internal view returns (uint256) {
+        if (_rate == 0) {
+            return 0;
+        }
+
+        // base_balance = base_balance * (1 + r * t)
+        uint256 timeElapsed = (block.timestamp - _lastUpdated).divDown(
+            365 days
+        );
+        uint256 accrued = totalPooledEther.mulDown(_rate.mulDown(timeElapsed));
+        return accrued;
+    }
+
+    function calculateTVLs()
+        public
+        view
+        override
+        returns (uint256[][] memory, uint256[] memory, uint256)
+    {
+        uint256[][] memory operator_tokens_tvls;
+        uint256[] memory operator_tvls;
+        uint256 tvl = getTotalPooledEther();
+        return (operator_tokens_tvls, operator_tvls, tvl);
+    }
+
+    function depositETH() external payable {
+        revert("depositETH: Not Implemented");
+    }
+
+    function ezETH() external view returns (address) {
+        return address(this);
+    }
+
+    function renzoOracle() external view returns (address) {
+        return address(this);
+    }
+
+    // Renzo Oracle Functions //
+
+    function lookupTokenValue(
+        IERC20, //_token,
+        uint256 //_balance
+    ) external pure returns (uint256) {
+        revert("lookupTokenValue: Not Implemented");
+    }
+
+    function lookupTokenAmountFromValue(
+        IERC20, // _token,
+        uint256 // _value
+    ) external pure returns (uint256) {
+        revert("lookupTokenValue: Not Implemented");
+    }
+
+    function lookupTokenValues(
+        IERC20[] memory, // _tokens,
+        uint256[] memory // _balances
+    ) external pure returns (uint256) {
+        revert("lookupTokenValue: Not Implemented");
+    }
+
+    function calculateMintAmount(
+        uint256, //  _currentValueInProtocol,
+        uint256, // _newValueAdded,
+        uint256 // _existingEzETHSupply
+    ) external pure returns (uint256) {
+        revert("lookupTokenValue: Not Implemented");
+    }
```

Comment:
This is used in `_convertToShares`. `_convertToShares` is used in `_withdraw`, so I think this will cause problems for us.